src/components/ui/UiMap.vue

```diff
@@ -1,10 +1,10 @@
 <script setup lang="ts">
-import { MapCollection } from '@vcmap/core'
+import { VcsApp } from '@vcmap/core'
 import { ref, watch } from 'vue'
 
 const props = defineProps({
   map: {
-    type: MapCollection,
+    type: VcsApp,
```

Comment:
perhaps also change the props's name to vcsApp or something more descriptive (map is obviously not)